src/test/java/edu/ucla/library/services/metadata/MetadataSetterTest.java

```diff
@@ -107,6 +111,25 @@ public void testGetMetaWithFile() throws Exception {
         assertTrue(Files.exists(FileSystems.getDefault().getPath(OUTPUT_PATH + CSV_NAME)));
     }
 
+    /**
+     * Tests populating Format.extent.
+     */
+    @Test
+    public void testPopulateFormatExtentField() throws Exception {
+        final int statusCode = catchSystemExit(() -> {
+            MetadataSetter.main(new String[] { CSV_PATH + CSV_NAME, MEDIA_PATH, FFMPEG_PATH, OUTPUT_PATH });
+        });
+
+        final String formattedDuration = "12m 37s";
+        final CSVReader reader = new CSVReader(new FileReader(
+                                 FileSystems.getDefault().getPath(OUTPUT_PATH + CSV_NAME).toFile()));
+        final List<String[]> rows = reader.readAll();
+        final CsvHeaders headers = new CsvHeaders(rows.get(0));
+
+        reader.close();
```

Comment:
Using the try-with-resources syntax would save you from having to call the close() yourself. Not a blocker though. Just noting it so I train myself to use it more too.

Reply:
Switched to try-with-resources.